{{cookiecutter.project_slug}}/tests/test_{{cookiecutter.project_slug}}.py

```diff
@@ -26,30 +26,26 @@
 
 
 {% if cookiecutter.use_pytest == 'y' -%}
-class Test{{ cookiecutter.project_slug|title }}(object):
+@pytest.fixture
+def some_fixture():
+    pass
```

Comment:
The pytest fixture isn't really a replacement of the setup and teardown methods of unittest, I believe it's more of an improvement that changes how tests are created. If a user decided to use a pytest fixture as the setup and teardown methods in their code, they would use yield as you mentioned, and the code would continue to run in a test regardless of exception status.

I see your point about the test function testing the fixture, rather than something else independently. 

So instead of:
```
def test_response(response):
    """Sample pytest test function with the pytest fixture as an argument.
    """
    # assert response.status_code == 200
    pass
```

We could do something like:
```
def test_content(response):
    """Sample pytest test function with the pytest fixture as an argument.
    """
    # from bs4 import BeautifulSoup
    # assert 'class' in BeautifulSoup(response.content)
    pass
```